Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-rails (3.8.2)
+    rspec-rails (3.8.3)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       railties (>= 3.0)
       rspec-core (~> 3.8.0)
       rspec-expectations (~> 3.8.0)
       rspec-mocks (~> 3.8.0)
       rspec-support (~> 3.8.0)
-    rspec-support (3.8.2)
-    rubocop (0.74.0)
+    rspec-support (3.8.3)
+    rubocop (0.78.0)
       jaro_winkler (~> 1.5.1)
       parallel (~> 1.10)
       parser (>= 2.6)
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.3.2)
+    rubocop-rails (2.4.0)
```

Comment:
### [rubocop-rails](https://docs.rubocop.org/projects/rails)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.2 → 2.4.0

[_[change-log](https://github.com/rubocop-hq/rubocop-rails/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop-hq/rubocop-rails/)_]





<details>
<summary>Commits</summary>

A change of **41** commits. See the full changes on [the compare page](https://github.com/rubocop-hq/rubocop-rails/compare/v2.3.2...v2.4.0).

These are the first 10 commits:
- (3b3962c) [Replace use of strip_indent with squiggly heredoc](https://github.com/rubocop-hq/rubocop-rails/commit/3b3962c73da36bd273dd8222e6e45e35bf4966e5)
- (9adacb9) [Merge pull request #124 from eugeneius/squiggly_heredoc](https://github.com/rubocop-hq/rubocop-rails/commit/9adacb9c1416a8943fa787409e093d3c27b704d3)
- (f223f0c) [Add ApplicationController and ApplicationMailer cops](https://github.com/rubocop-hq/rubocop-rails/commit/f223f0c3d3ef25ab356194b2f88678dbb651dbd9)
- (2058629) [Merge pull request #123 from eugeneius/application_controlle…](https://github.com/rubocop-hq/rubocop-rails/commit/20586297d465867e7e19176a98e606516c6b81d3)
- (f7287a2) [[Fix #120] Fix save-in-conditional checking in `Rails/SaveBa…](https://github.com/rubocop-hq/rubocop-rails/commit/f7287a2d18acde6403b3ce7e2b84a8b44466c06b)
- (ee0176b) [Merge pull request #121 from jas14/fix-savebang-in-condition…](https://github.com/rubocop-hq/rubocop-rails/commit/ee0176b01c1d82d484335b93a8b36954e2032bc8)
- (3e6cde2) [Add `require 'English'` to enable `$RS` global variable](https://github.com/rubocop-hq/rubocop-rails/commit/3e6cde2350d4e7b195f767fe85be269ea286f108)
- (80fcf14) [Merge pull request #129 from pocke/require-English](https://github.com/rubocop-hq/rubocop-rails/commit/80fcf1475a59c2aa75d16e01f9e34b8006e212be)
- (21a3285) [presence cop: fix [] for ternary expr other hand](https://github.com/rubocop-hq/rubocop-rails/commit/21a32858d45bf660cd2d89d250df2b6a34809515)
- (3421da1) [Restyle example formatting for `Rails/SafeNavigation`](https://github.com/rubocop-hq/rubocop-rails/commit/3421da1e64b7d54a21b8bb7d84376d75da1fa0fe)


</details>